### ISSUE TITLE
check `finalize-classification` and `finalize-regression` in all models

### ISSUE THREAD

**Issue by behrica** [scicloj/noj#205]
finalize-classification
finalize-regression
tech.v3.dataset.modelling/probability-distributions->label-column

have eventual different copies in each model,maybe doing the same or not

**Comment by behrica** [comment]
smile: https://github.com/scicloj/scicloj.ml.smile/blob/a09266e4e423f0550425a5c55327271dd7f48a37/src/scicloj/ml/smile/model.clj#L52
xgboost: https://github.com/scicloj/scicloj.ml.xgboost/blob/4e9f3097413eeac33f25999203420f4f98b64805/src/scicloj/ml/xgboost/model.clj#L25
sklearn-clj:  does its own thing
tribuo:  does its own thing

**Comment by behrica** [comment]
smile + xgboost have the same code.

Both assume that the "order" of categories in the categorical map, matches "order of columns" of the probability distribution returned by the model
